In [2]:
import pandas as pd

In [5]:
df = pd.read_csv(r'C:\Users\SAHITHI KODUMURI\Documents\Fraud-detection-baseline\data\creditcard.csv')

In [6]:
df.shape

(284807, 31)

In [7]:
df.isnull().sum()

Time      0
V1        0
V2        0
V3        0
V4        0
V5        0
V6        0
V7        0
V8        0
V9        0
V10       0
V11       0
V12       0
V13       0
V14       0
V15       0
V16       0
V17       0
V18       0
V19       0
V20       0
V21       0
V22       0
V23       0
V24       0
V25       0
V26       0
V27       0
V28       0
Amount    0
Class     0
dtype: int64

In [8]:
df['Class'].value_counts()

Class
0    284315
1       492
Name: count, dtype: int64

## Initial Observations of the dataset
1) The data set has 2,84,315 rows, 31 columns and no missing values.
2) Severe class imbalance - fraud is only ~0.17% of total transactions(492 out of 284315).
3) Accuracy will be a misleading metric in this case. So, precision, recall and F1 matter more in this case.